pandas/_libs/tslibs/strptime.pxd

```diff
@@ -0,0 +1,5 @@
+from numpy cimport int64_t
+
+
+cpdef bint format_is_iso(str f)
```

Comment:
Is this only made cpdef because it appears in a test directly? I think would be better to have as `cdef` if not used in the codebase directly

Reply:
that's right! i've made a `def` wrapper around it to be used in testing, is that OK?

https://github.com/pandas-dev/pandas/blob/33dec0340e9ad3d9b88caa3178d9260e7eb7bfa0/pandas/_libs/tslibs/strptime.pyx#L73-L75

Reply:
Seems reasonable